hack/release.sh

```diff
@@ -19,17 +19,16 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/release.sh
 # Yaml files to generate, and the source config dir for them.
 declare -A COMPONENTS
 COMPONENTS=(
-  ["sources.yaml"]="config"
+  ["release.yaml"]="config"
   ["gcppubsub.yaml"]="contrib/gcppubsub/config"
   ["message-dumper.yaml"]="config/tools/message-dumper"
 )
 readonly COMPONENTS
 
+# Yaml files that are combinations of the above components. Release files must
+# not have the same name as any component files.
 declare -A RELEASES
 RELEASES=(
-  ["release.yaml"]="sources.yaml"
```

Comment:
1. This array is empty, it can be removed.
1. Now the loop at line 44 won't be executed, but it wasn't removed from the code.

Is it intended?

Reply:
Synced with Grant, this is intended. Dead code cleanup in https://github.com/knative/eventing-sources/pull/236

Reply:
This is intended. I didn't remove the array or the loop so that this release.sh looks similar to [the eventing one](https://github.com/knative/eventing/blob/24e3c020eeb46ee18650f47ae0c297228bbfb4db/hack/release.sh).

#236 tracks this.